roles/dns/templates/named.conf.zones

```diff
@@ -20,4 +20,19 @@ zone "buildbot.net" {
     };
 };
 
+zone "224-255.128-255.10.211.140.in-addr.arpa" {
+    type master;
+    file "{{ namedb_dir }}/master/224-255.128-255.10.211.140.in-addr.arpa";
+    allow-transfer {
+        "buildbot.net-upstream";
+    };
+    notify explicit;
+    also-notify {
+        173.244.206.26;     /* a.transfer.buddyns.com */
```

Comment:
can we actually do that?  I thought the only upstream for us is the one responding for `10.211.140.in-addr.arpa.`...